src/autofill.js

```diff
@@ -3,6 +3,18 @@
         if (!window.isSecureContext) return
 
         const listenForGlobalFormSubmission = require('./Form/listenForFormSubmission')
+        const {isAndroid} = require('./autofill-utils')
+        const {processConfig} = require('@duckduckgo/content-scope-scripts/src/apple-utils')
+
+        if (!isAndroid) {
+            // @ts-ignore - variable populated during injection
+            const privacyConfig = processConfig($CONTENT_SCOPE$, $USER_UNPROTECTED_DOMAINS$, $USER_PREFERENCES$)// eslint-disable-line no-undef
```

Comment:
Can we extract these variables like we did for other similar instances like [here](https://github.com/duckduckgo/duckduckgo-autofill/blob/aec2ae7f48473d52dd92dfd7f40f5c044fe5d39e/src/autofill-utils.js#L3)? You can literally copy the same pattern used there. Ping me on MM if it's not clear.